v3/src/components/case-table/attribute-menu/attribute-menu-list.tsx

```diff
@@ -18,34 +19,50 @@ export const AttributeMenuList = forwardRef<HTMLDivElement, IProps>(
   const toast = useToast()
   const data = useDataSetContext()
   const caseMetadata = useCaseMetadata()
-  const { isOpen, onOpen, onClose } = useDisclosure()
+  const {
+    isOpen: isEditAttributePropsOpen, onOpen: onEditAttributePropsOpen, onClose: onEditAttributePropsClose
+  } = useDisclosure()
+  const {
+    isOpen: isEditFormulaOpen, onOpen: onEditFormulaOpen, onClose: onEditFormulaClose
+  } = useDisclosure()
```

Comment:
It took me a few minutes to figure out why `useDisclosure()` can be used multiple times this way. Perhaps a comment like the following would be useful:
```suggestion
  // each use of useDisclosure() maintains its own state and callbacks so they can be used for independent dialogs
  const {
    isOpen: isEditAttributePropsOpen, onOpen: onEditAttributePropsOpen, onClose: onEditAttributePropsClose
  } = useDisclosure()
  const {
    isOpen: isEditFormulaOpen, onOpen: onEditFormulaOpen, onClose: onEditFormulaClose
  } = useDisclosure()
```

Reply:
I see. I've committed your comment + replaced destructuring assignment with `const formulaModal` and `const attributePropsModal` as it might be more readable and easier to use than separate variables.